### ISSUE TITLE
System ui (status bar and navigation bar) doesn't hide automatically.

### ISSUE THREAD

**Issue by makindotcc** [pichillilorenzo/flutter_inappwebview#202]
## Environment

**App version:** ? you mean this plugin version? 2.0.1
**Android version:** 9 (pixel experience rom) and 7.1.1 android emulator and 7.1 samsung official rom
**Device information:**  xiaomi redmi 4x, my pc, samsung galaxy s6
**Flutter version:** 1.9.1+hotfix.6

## Description

**Expected behavior:** 
Hide the system ui after a short interval.

**Current behavior:** 
System ui doesn't hide automatically.

## Steps to reproduce

1. Create webview:
```dart
    InAppWebView(
      initialUrl: "https://www.youtube.com/embed/a3ICNMQW7Ok?autoplay=1",
      onLoadStop: _onPageFinished,
      onWebViewCreated: _onWebViewCreated,
      initialOptions: InAppWebViewWidgetOptions(
        inAppWebViewOptions: InAppWebViewOptions(
          userAgent: "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/78.0.3904.97 Safari/537.36",
          mediaPlaybackRequiresUserGesture: false,
          javaScriptEnabled: true,
          useShouldOverrideUrlLoading: true,
        ),
      ),
      initialHeaders: {
        "Accept-Language": "pl"
      },
      shouldOverrideUrlLoading: _onShouldOverrideUrlLoading,
    )
```
2. Enter to fullscreen mode
3. Swipe down from top to show system ui

## Video
https://youtu.be/IKrwPBZwPSU

## Stacktrace/Logcat
nothing is logged

**Comment by pichillilorenzo** [comment]
Now it should be fixed in the new version **2.1.0**!

**Comment by makindotcc** [comment]
Thank you for quick fix. Works for me too :)
